content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -60,13 +62,13 @@ spec:
     image: nginx
 ```
 
-Then add a nodeSelector like so:
+Then add a nodeSelector using the following commands:
 
 {{< codenew file="pods/pod-nginx.yaml" >}}
 
 When you then run `kubectl apply -f https://k8s.io/examples/pods/pod-nginx.yaml`,
-the Pod will get scheduled on the node that you attached the label to. You can
-verify that it worked by running `kubectl get pods -o wide` and looking at the
+the Pod will get scheduled on the node that you attached the label to.
+You can verify that if  it worked by running the  `kubectl get pods -o wide` and by looking at the
```

Comment:
To verify that it worked, run `kubectl get pods -o wide`. Check that the pod is assigned to the correct node.